firmware/application/apps/ui_pocsag_tx.cpp

```diff
@@ -80,8 +80,7 @@ bool POCSAGTXView::start_tx() {
     progressbar.set_max(total_frames);
 
     transmitter_model.set_rf_amp(true);
-    transmitter_model.set_lna(40);
```

Comment:
These didn't actually do anything. Using tx_gain instead.

Reply:
Fully agree with that , this is for RX.